src/main/java/no/sikt/nva/pubchannels/dataporten/Journal.java

```diff
@@ -0,0 +1,78 @@
+package no.sikt.nva.pubchannels.dataporten;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import java.net.URI;
+import no.sikt.nva.pubchannels.Immutable;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+/* default */ final class Journal implements Immutable {
+    private static final String YEAR_FIELD = "Year";
+    private static final String EISSN_FIELD = "Eissn";
+    private static final String PID_FIELD = "Pid";
+    private static final String LEVEL_FIELD = "Level";
+    private static final String PISSN_FIELD = "Pissn";
+    private static final String NAME_FIELD = "Name";
+    private static final String KURL_FIELD = "KURL";
+
+    @JsonProperty(YEAR_FIELD)
+    private final String year;
+    @JsonProperty(EISSN_FIELD)
+    private final String eissn;
+    @JsonProperty(PID_FIELD)
+    private final String pid;
+    @JsonProperty(LEVEL_FIELD)
+    private final String level;
+    @JsonProperty(PISSN_FIELD)
+    private final String pissn;
+    @JsonProperty(NAME_FIELD)
+    private final String name;
+    @JsonProperty(KURL_FIELD)
+    private final URI kurl;
```

Comment:
I'd probably prefer the fields to be named according to what they are since pid, eissen, pissn, etc. map easily to existing concepts.

Reply:
fixed